### PR TITLE
hashicorp-vault include vaultKVUsePrefix setting

### PR DESCRIPTION
**Pull request by Banchio**
as per pull request https://github.com/dapr/components-contrib/pull/929

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [ X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

Other fields should not be applicable as I only added a new parameter.

## Description

Added a setting included in the PR referenced above

## Issue reference

https://github.com/dapr/components-contrib/issues/925

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ spec:
     value : "[path_to_file_containing_token]"
   - name: vaultKVPrefix # Optional. Default: "dapr"
     value : "[vault_prefix]"
+  - name: vaultKVUsePrefix # Optional. default: "true"
+    value: "true/false"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md]:
```suggestion
    value: "[true/false]"
```

File: daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md [hunk 2]
```diff
@@ -57,6 +59,7 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 | vaultTokenMountPath | Y | Path to file containing token | `"path/to/file"` |
 | vaultToken | Y | [Token](https://learn.hashicorp.com/tutorials/vault/tokens) for authentication within Vault.  | `"tokenValue"` |
 | vaultKVPrefix | N | The prefix in vault. Defautls to `"dapr"` | `"dapr"`, `"myprefix"` |
+| vaultKVUsePrefix | N | If false, vaultKVPrefix is forced to be empty | `"true"`, `"false"` |
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md]: Please provide more information on what this field is and does

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md (hunk 2 above)
Comment (Banchio) [reply daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md]: @AaronCrawfis thanks for the review, I hope I added a commit that clarify this argument and I should have added this commit to this pull request, let me know if I did everything correctly

**Review by AaronCrawfis — APPROVED** [review]

File: daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md (hunk 2 above)
Comment (AaronCrawfis) [reply daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md]: Looks great, thanks!

**Comment by Banchio** [comment]
Rcvd an email about this failure, does this prevent the merge? It is not a link I included in the PR

**Comment by AaronCrawfis** [comment]
> Rcvd an email about this failure, does this prevent the merge? It is not a link I included in the PR

Looks like a transient failure. I won't block merging this on it

**Comment by CodeMonkeyLeet** [comment]
@AaronCrawfis The code PR has been merged as part of components-contrib v1.3, so this should probably be in v1.3 for docs too

**Pull request merged**